laceworksdk/api/v2/agent_access_tokens.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""
+Lacework AgentAccessTokens API wrapper.
+"""
+
+from laceworksdk.api.crud_endpoint import CrudEndpoint
+
+
+class AgentAccessTokensAPI(CrudEndpoint):
+
+    def __init__(self, session):
+        """
+        Initializes the AgentAccessTokensAPI object.
+
+        :param session: An instance of the HttpSession class
+
+        :return AgentAccessTokensAPI object.
+        """
+
+        super().__init__(session, "AgentAccessTokens")
+
+    def create(self,
+               alias=None,
+               enabled=True,
+               **request_params):
+        """
+        A method to create a new AgentAccessTokens object.
+
+        :param alias: A string representing the object alias.
+        :param enabled: A boolean/integer representing whether the object is enabled.
+            (0 or 1)
+        :param request_params: Additional request parameters.
+            (provides support for parameters that may be added in the future)
+
+        :return response json
+        """
+
+        return super().create(
+            token_alias=alias,
+            token_enabled=int(bool(enabled)),
+            **request_params
+        )
+
+    def get(self,
+            id=None):
+        """
+        A method to get AgentAccessTokens objects.
+
+        :param id: A string representing the object ID.
+
+        :return response json
+        """
+
+        return super().get(
+            id=id
+        )
```

Comment:
this function does not seem to provide a lot of value here... why overwrite it?

Reply:
removed